portal-impl/test/unit/com/liferay/portal/util/FileImplTest.java

```diff
@@ -223,6 +234,171 @@ public void testStripSuffixWhenFileNameHasParenthesisAtStart() {
 			"()test.jsp", _fileImpl.stripParentheticalSuffix("()test.jsp"));
 	}
 
+	@Test
+	public void testUnzip() throws Exception {
+		_setUpForUnzipTests();
+
+		File f = _createZipFileTestZip();
```

Comment:
@jeffwu0724 Please always use "File file" rather than "File f".

Reply:
Hi Tina, I will fix this once the review is done, in case there are more bugs in this PR.
Thanks for letting me know!